client/css/style.css

```diff
@@ -978,6 +984,42 @@ background on hover (unless active) */
 	font-size: 14px;
 }
 
+#windows .header .topic-input {
+	color: inherit;
+	background: transparent;
+	border: 1px solid #cdd3da;
+	border-radius: 2px;
+	padding-right: 37px;
+	padding-left: 10px;
+	width: 100%;
+	height: 35px;
+	overflow: hidden;
+	font-size: 14px;
+	outline: none;
+}
+
+.topic-container .save-topic {
+	position: absolute;
+	top: 6px;
+	right: 0;
+}
+
+.topic-container .save-topic span {
+	font: normal normal normal 14px/1 FontAwesome;
```

Comment:
Look on line 249, we define the fontawesome font in the big selector there.

Reply:
This is fixed now.